CMakeLists.txt

```diff
@@ -199,6 +193,15 @@ install(FILES ${CMAKE_CURRENT_BINARY_DIR}/omplConfig.cmake
     ${CMAKE_CURRENT_BINARY_DIR}/omplConfigVersion.cmake
     DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/ompl/cmake
     COMPONENT ompl)
+install(TARGETS ompl
+    EXPORT omplExport
+    DESTINATION ${CMAKE_INSTALL_LIBDIR}
```

Comment:
Might consider changing to `CMAKE_INSTALL_DATAROOTDIR` which seems more common. find_package works either way.

Reply:
Done in 900ccc114d8739e817c9ac7f1268d076c2b99ef8, now it is consistent